test/test-recipes.js

```diff
@@ -12,7 +12,7 @@ var testUtil = require('./testUtil');
 // Helpers
 // --------------------------------------------------------------------
 
-function toGrammar(recipeString) {
+function toGrammarOrRecipe(recipeString) {
```

Comment:
Yikes! So obvious (it should have been `toGrammarOrSemantics`). But I will call it `makeRecipe`. Thanks :-)